go/common/sgx/ias/avr_test.go

```diff
@@ -14,6 +14,9 @@ func TestAVR(t *testing.T) {
 }
 
 func testAVRv2(t *testing.T) {
+	// TODO: Generate another testdata AVR with debug bit disabled.
+	unsafeAllowDebugEnclaves = true
```

Comment:
Unset this in a defer so that other tests that depend on verification being strict do the right thing.